### ISSUE TITLE
Class attributes naming rules

### ISSUE THREAD

**Issue by sobolevn** [PyCQA/pep8-naming#78]
I have occasionally found out that it is possible to use any case for naming class attributes:

```python
class Test(object):
    camelCase = 1
    snake_case = 2
    UPPER_CASE = 3

```

`flake8` and `pep8-naming` won't complain about it.

Since https://www.python.org/dev/peps/pep-0008 does not say a word about class attributes naming style, I believe that it might be a hot discussion. 

In my opinion class attributes should be using `snake_case` by default.
What do you think? Thanks!

**Comment by lordmauve** [comment]
I would read `UPPER_CASE` as indicating a class-level constant. For example, [enum.Enum](https://docs.python.org/3/library/enum.html) would typically be defined with upper-case names.

I'd like to see warnings about `camelCase` class attributes, however.

**Comment by sigmavirus24** [comment]
I agree with @lordmauve that `UPPER_CASE` should not be forbidden. `camelCase` does make sense to warn about though. I'd review a PR that implements that change.

**Comment by 5j9** [comment]
I can't test right now, but I believe I implemented this nearly 4 months ago. (https://github.com/PyCQA/pep8-naming/commit/16793542023da874242101693c13f64d3536bbdf)

**Comment by sobolevn** [comment]
It is not published yet, I guess.

**Comment by sigmavirus24** [comment]
Excellent. Let's prepare a release then @5j9

**Comment by 5j9** [comment]
@sigmavirus24, is there anything I can do in preparation for a new release?

**Comment by sigmavirus24** [comment]
I think this was released in 0.8.0
